### PR TITLE
[cherry pick 6.3] Include gfx12 and gfx1151 in default architecture list for rmake.py

### PR DESCRIPTION
**Pull request by NguyenNhuDi**

### CODE REVIEW

**Review by umfranzw — APPROVED** [review]
Looks good to me. The build failures look like they are unrelated to this change.

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,10 +10,11 @@ Full documentation for hipCUB is available at [https://rocm.docs.amd.com/project
 
 ### Changed
 
+* Updated the default value for the `-a` argument from `rmake.py` to `gfx906:xnack-,gfx1030,gfx1100,gfx1101,gfx1102,gfx1151,gfx1200,gfx1201`.
```
Comment (spolifroni-amd) [inline CHANGELOG.md:13]:
```suggestion
* Updated the default value for the `-a` argument from `rmake.py` to `gfx906:xnack-,gfx1030,gfx1100,gfx1101,gfx1102,gfx1151,gfx1200,gfx1201`.
```

Comment (spolifroni-amd) [inline CHANGELOG.md]: It looks like the new default value is `default_gpus`. Can you clarify if this is the same as `gfx906:xnack-,gfx1030,gfx1100,gfx1101,gfx1102,gfx1151,gfx1200,gfx1201`. Thanks!

```suggestion
* Changed the default value of `rmake.py -a` to `gfx906:xnack-,gfx1030,gfx1100,gfx1101,gfx1102,gfx1151,gfx1200,gfx1201`.
```

**Review by spolifroni-amd — CHANGES_REQUESTED** [review]
I have one comment that needs to be addressed.

File: CHANGELOG.md (hunk 1 above)
Comment (NguyenNhuDi) [reply CHANGELOG.md]: It was the same but without gfx1200,gfx1201 and gfx1151

Comment (spolifroni-amd) [reply CHANGELOG.md]: Ah OK! Let me rewrite this then.

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,10 +10,11 @@ Full documentation for hipCUB is available at [https://rocm.docs.amd.com/project
 
 ### Changed
 
+* Changed the default value of `rmake.py -a` to `gfx906:xnack-,gfx1030,gfx1100,gfx1101,gfx1102,gfx1151,gfx1200,gfx1201`.
```
Comment (spolifroni-amd) [inline CHANGELOG.md]:
```suggestion
* Changed the default value of `rmake.py -a` to `default_gpus`. This is equivalent to `gfx906:xnack-,gfx1030,gfx1100,gfx1101,gfx1102,gfx1151,gfx1200,gfx1201`.
```

**Review by spolifroni-amd — CHANGES_REQUESTED** [review]
rewrote the rmake.py -a changelog point to make the change clearer,

**Review by spolifroni-amd — APPROVED** [review]

**Pull request merged**